torch/_inductor/fx_passes/pre_grad.py

```diff
@@ -48,7 +48,7 @@ def lazy_init():
     from . import efficient_conv_bn_eval, split_cat  # noqa: F401  # noqa: F401
 
     if config.is_fbcode():
-        from .fb import split_cat as split_cat_fb  # type: ignore[import]  # noqa: F401
+        from . import fb  # type: ignore[import]  # noqa: F401
```

Comment:
Noob q: how does importing this within the function help the outside code? Are there some persistent side effects from importing?

Reply:
yes within those python modules (like `split_cat.py`), there's a global function registered with `@register_graph_optimization(...)`. So just importing is enough.